docs/cody/clients/cody-with-sourcegraph.mdx

```diff
@@ -1,60 +1,66 @@
+
+The chat interface for Cody on the web is similar to the one you get with the [VS Code](/cody/clients/install-vscode#chat) editor extension. If you click the Cody chat from the top header, you will see a **Start new chat +** button in the bottom left sidebar with the chat editor. All your previous chats appear in this left sidebar. To delete any chat, hover over the chat and click the **Delete** icon.
+
+The chat interface with your Code Search queries is operated per chat. You cannot run multiple chats and store them in parallel. A new chat window opens whenever you click the Cody button from the query editor or the top header.
+
+## LLM Selection
+
+Sourcegraph.com users with Cody **Free** and **Pro** can choose from a list of supported LLM models for a chat. Claude Sonnet 3 is the default LLM model, but users can select the LLM of their choice from the drop-down menu.
+
+![llm-select-web](https://storage.googleapis.com/sourcegraph-assets/Docs/llm-select-web-0724.jpg)
+
+Users on an Enterprise Sourcegraph instance do not have the option to choose an LLM model. Their site admin will configure the default LLM model for chat.
 
-To view how Cody works in the web, open **Cody Web Chat**. Here, you can ask Cody general coding questions or questions about the repositories you care about.
+## Selecting Context with @-mentions
 
-You can view the default **Ask Cody** chat interface on the right sidebar. Once connected, Cody automatically gains context to your connected codebase to help you provide contextually-aware and accurate answers to your questions. There are also example questions that you can use to get started.
+Cody chat with a Sourcegraph.com instance allows you to add files and symbols as context in your messages.
 
-When using Cody chat in the web, you can select up to **10 repositories** to use as context for your question. If you’re logged in to Sourcegraph.com, these can be any **10 open source repositories** that Sourcegraph indexes. If you are an enterprise customer logged in to your company’s Sourcegraph instance, these can be **any 10 repositories** (both private or public) indexed by your admin.
+- Type `@file` and then a filename to include a file as a context
+- Type `@` and you can also add **Symbols**, **Files**, and **Repositories** as context
 
-You can also chat with Cody in the sidebar of any project you have open. Use Sourcegraph search to find a repo and click the **Ask Cody** button to start.
+### Context retrieval
 
-    <video width="1920" height="1080" loop playsInline controls style={{ width: '100%', height: 'auto' }}>
-          <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/search-cody-chat.mp4" type="video/mp4" />
-          </video>
+When you start a new Cody chat inside the search query window, the chat input window opens with a default `@-mention` context chip for all the context it intends to use. This context is based on your current repository and current file.
```

Comment:
I'd also add that by default when you open the side panel Cody chat, this chat gets this repository as context even without having direct mentions of files and repositories.

Also, later this chat from blob UI will be available to you on the standalone chat as well.